code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -109,6 +109,7 @@
 		H.underwear = "Nude"
 		H.update_body()
 		ADD_TRAIT(H, TRAIT_PRIMITIVE, ROUNDSTART_TRAIT)
+		H.remove_language(/datum/language/common)
```

Comment:
I would probably suggest a better way of doing this would be going to `/datum/language_holder/lizard/ash` and setting `understood_languages` and `spoken_languages` to only have `/datum/language/draconic = list(LANGUAGE_ATOM)`

I'll merge it after that's done.

Reply:
That doesn't work. Should I add galactic common to blocked languages?

Reply:
That may be necessary as well. If you can't get it to work properly like that I'll go back to the drawing board and if it turns out I have no clue what I'm talking about I'll merge this as-is.

Reply:
Well, adding it to blocked does work, but it also prevents them from ever learning it. I like the idea of an ashie occasionally getting their hands on a book of babel, so I'm not a huge fan of this solution.

Reply:
The full answer is "it's complex" - Short answer is turns out that the languages a mob can speak are a mix of the mob's languages and the mind's languages. Pretty much every mind understands common. It's blegh, but I'll merge this as-is.